modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/Policy.java

```diff
@@ -99,4 +99,20 @@ public void deletePolicy(@PathParam("id") int policyId) throws MDMAPIException {
 			throw new MDMAPIException(error, e);
 		}
 	}
+
+	@GET
+	@Path("count")
+	public int getPolicyCount() throws MDMAPIException {
+		int policyCount = 0;
+		PolicyManagerService policyManagementService = MDMAPIUtils.getPolicyManagementService();
+		try {
+			PolicyAdministratorPoint pap = policyManagementService.getPAP();
+			policyCount = pap.getPolicyCount();
+			return policyCount;
```

Comment:
There's no need to have a poolicyCount variable here. You can simple return pap.getPolicyCount();